mindsdb/interfaces/database/integrations.py

```diff
@@ -234,6 +235,16 @@ def create_handler(self, name: str = None, handler_type: str = None,
                 ViewController(),
                 company_id=company_id
             )
+        elif handler_type == 'lightwood':
```

Comment:
Probably better to do a generic "predictive handler" check and enforce all of them (lightwood included) to conform to the API defined by `ModelInterface()`. Not sure how to pass the class type instead of the handler name, though.